modules/private-clusters-setting-dns-private.adoc

```diff
@@ -6,11 +6,17 @@
 [id="private-clusters-setting-dns-private_{context}"]
 = Setting DNS to private
 
-After you deploy a cluster, you can modify its DNS to use only a private zone.
+After you deploy a cluster, you can modify its DNS to use only a private zone, by modifying the `DNS` custom resource (CR).
+Because the Ingress Controller consults the `DNS` CR definition when it creates `Ingress` objects, when you create or modify `Ingress` objects, only private records are created. This secures the cluster by preventing public DNS records from being created.
```

Comment:
Maybe something like this, if @Miciah agrees with my re-interpretation:
```suggestion
After you deploy a cluster, you can modify its DNS to use only a private zone, meaning that any DNS records that are created are not published to public DNS servers.  This effectively keeps knowledge of the DNS records isolated for internal users.  This should be done when you configure the cluster to be private, or if you never want DNS records to be publicly resolvable.
```

Reply:
Yes, that wording is more accurate than the original text.

Reply:
Awesome, thanks folks 🙏🏻